protocol/proto/openyolo.proto

```diff
@@ -23,42 +23,45 @@ option java_outer_classname = "Protobufs";
 
 message CredentialRetrieveRequest {
     // at least one authMethod required
-    repeated string authMethods = 1;
-    map<string, TokenRequestInfo> supportedTokenProviders = 2;
-    map<string, bytes> additionalProps = 3;
+    repeated AuthenticationMethod authMethods = 1;
+
```

Comment:
Since I'm rapidly iterating on these changes I'll ignore this for now, we can check once all the changes are in that we match the protos in the spec.